@lingosta/app/app/globals.css

```diff
@@ -2,7 +2,7 @@
 @tailwind components;
 @tailwind utilities;
 
-:root {
+/* :root {
```

Comment:
If you don't plan on using this, just remove the block :) It's all version controlled anyways so you can reference it if you need it later. Keeps our code clean from becoming a bunch of commented blocks.